ui/src/services/alertHooks.js

```diff
@@ -0,0 +1,69 @@
+//@flow
+import { useAlerts } from '../containers/AlertProvider';
+import { getNodes } from './k8s/core.js';
+import { getMetalk8sV1alpha1VolumeList } from './k8s/Metalk8sVolumeClient.generated';
+import type { FilterLabels, Alert } from './alertUtils';
+import { getHealthStatus } from './alertUtils';
+
+type Query = { cacheKey: string, queryFn: Function, options?: Object };
+
+export const getNodesAlertName = (): string[] => {
+  return ['NodeAtRisk', 'NodeDegraded'];
+};
+
+export const getVolumesAlertName = (): string[] => {
+  return ['VolumeAtRisk', 'VolumeDegraded'];
+};
+
+export const getNetworksAlertName = (): string[] => {
+  return ['ControlPlaneNetworkDegraded', 'WorkloadPlaneNetworkDegraded'];
+};
+
+export const getServicesAlertName = (): string[] => {
+  return ['PlatformServicesAtRisk', 'PlatformServicesDegraded'];
+};
+
+/*
+The critical and warning alerts may active at the same time.
+how to use:
+  const nodesAlert = useHighestSeverityAlert({
```

Comment:
Nit : You may want to use jsdoc notation for this kind of comments to benefit from it in IDE autocompletion 

The name of the functions doesn't match the implementation :
```suggestion
  const nodesAlert = useHighestSeverityAlerts({
```